src/Header/Nav/__snapshots__/Nav.test.jsx.snap

```diff
@@ -232,7 +232,9 @@ exports[`Nav internalServices Matches snapshot with sub links for selected inter
                   id="navlink-internal1"
                   onClick={[Function]}
                 >
-                  <span>
+                  <span
```

Comment:
This snapshot looks wrong to me

Reply:
updated the snapshot and fixed a couple of failing tests